Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml

```diff
@@ -1,116 +1,24 @@
 - type: vendingMachineInventory
   id: ClothesMateInventory
   startingInventory:
-    ClothingBackpack: 5
-    ClothingBackpackDuffel: 5
-    ClothingBackpackSatchel: 3
-    ClothingBackpackSatchelLeather: 2
```

Comment:
Keep this backpack and then the PR is fine; just to allow for maximum storage customisation :)

Reply:
Done.